packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/BillingHistory.tsx

```diff
@@ -29,6 +33,9 @@ const BillingHistory = () => {
       >
         <Trans>Billing history</Trans>
       </Typography>
+      {isPendingInvoice && <Typography>
+        Please complete payment of the following outstanding invoices in order to avoid account suspension.
```

Comment:
I think that Trans flew away somehow
```suggestion
        <Trans>Please complete payment of the following outstanding invoices in order to avoid account suspension</Trans>
```